static/src/stylesheets/module/experiments/_embed.scss

```diff
@@ -157,4 +157,32 @@ a[href].contributions__contribute.contributions__contribute--epic {
     max-width: 620px;
 }
 
+.contributions__epic-footer {
+    margin-top: $gs-baseline;
+
+    h2 {
+        @include fs-header(5);
+        font-size: 32px;
+        line-height: 36px;
+    }
+}
+
+.contributions__epic--feb {
```

Comment:
this styling is specific to the campaign. The class is added per-variant from the sheet

Reply:
I'd actually call this one `--moment` so we can leave it in the codebase afterwards - I suspect the next moment will have very similar styling (based on the last two!)